### ISSUE TITLE
Test cannot directly copy and run cmd that is printed in console

### ISSUE THREAD

**Issue by llxia** [eclipse/openj9#1375]
Running test in build, we will get console output as following:
```
===============================================
Running test memoryCategories_0 ...
===============================================
test with NoOptions
{ mkdir -p "/tmp/bld_380527/memoryCategories_0"; \
		cd "/tmp/bld_380527/memoryCategories_0"; \
		export LIBPATH="/bluebird/builds/bld_380527/sdk/xa6480/jre/bin/../lib/default:/bluebird/builds/bld_380527/sdk/xa6480/jre/bin/../lib/amd64/default:/bluebird/builds/bld_380527/sdk/xa6480/jre/bin/j9vm:"; \
		"/bluebird/builds/bld_380527/sdk/xa6480/jre/bin/java"  -Xnocompressedrefs  \
	-cp "/bluebird/builds/bld_380527/jvmtest/test/SE80/TestConfig/resources:/bluebird/builds/bld_380527/jvmtest/test/SE80/TestConfig/lib/testng.jar:/bluebird/builds/bld_380527/jvmtest/test/SE80/TestConfig/lib/jcommander.jar:/bluebird/builds/bld_380527/jvmtest/test/SE80/Java8andUp/GeneralTest.jar" \
	org.testng.TestNG -d "/tmp/bld_380527/memoryCategories_0" "/bluebird/builds/bld_380527/jvmtest/test/SE80/Java8andUp/testng.xml" \
	-testnames memoryCategories \
	-groups level.extended \
	-excludegroups d.*.linux_x86-64,d.*.arch.x86,d.*.os.linux,d.*.bits.64,d.*.generic-all; \
	if [ $? -eq 0 ] ; then echo "memoryCategories_0""_PASSED"; else perl "-I/bluebird/builds/bld_380527/../../javatest/HEAD_379871/test/lib/perl" -mResultStore::Uploader -e "ResultStore::Uploader::upload('.',380527,532827023,'vmfarm.ottawa.ibm.com:31','results-532827023')"; echo; echo "memoryCategories_0""_FAILED"; fi; } 2>&1 | tee -a "/tmp/bld_380527/TestTargetResult"
[IncludeExcludeTestAnnotationTransformer] [INFO] exclude file is /bluebird/builds/bld_380527/jvmtest/test/SE80/TestConfig/resources/excludes/current_exclude_SE80.txt
...
... TestNG 6.10 by C?dric Beust (cedric@beust.com)
...
```
Ideally, we should be able to run the test by just run the cmd directly. (stated in test readme). 
```
mkdir -p "/tmp/bld_380527/memoryCategories_0"; \
		cd "/tmp/bld_380527/memoryCategories_0"; \
		export LIBPATH="/bluebird/builds/bld_380527/sdk/xa6480/jre/bin/../lib/default:/bluebird/builds/bld_380527/sdk/xa6480/jre/bin/../lib/amd64/default:/bluebird/builds/bld_380527/sdk/xa6480/jre/bin/j9vm:"; \
		"/bluebird/builds/bld_380527/sdk/xa6480/jre/bin/java"  -Xnocompressedrefs  \
	-cp "/bluebird/builds/bld_380527/jvmtest/test/SE80/TestConfig/resources:/bluebird/builds/bld_380527/jvmtest/test/SE80/TestConfig/lib/testng.jar:/bluebird/builds/bld_380527/jvmtest/test/SE80/TestConfig/lib/jcommander.jar:/bluebird/builds/bld_380527/jvmtest/test/SE80/Java8andUp/GeneralTest.jar" \
	org.testng.TestNG -d "/tmp/bld_380527/memoryCategories_0" "/bluebird/builds/bld_380527/jvmtest/test/SE80/Java8andUp/testng.xml" \
	-testnames memoryCategories \
	-groups level.extended \
	-excludegroups d.*.linux_x86-64,d.*.arch.x86,d.*.os.linux,d.*.bits.64,d.*.generic-all;
```

However, by just running the cmd directly, user will get error:
```
[IncludeExcludeTestAnnotationTransformer] [INFO] exclude file is null
[TestNG] [ERROR]
Cannot instantiate class org.openj9.test.util.IncludeExcludeTestAnnotationTransformer
```

**Comment by llxia** [comment]
@sophia-guo , could you take a look at this? Thanks.

**Comment by llxia** [comment]
If there is more env variables need to set, maybe we need to update readme.

**Comment by smlambert** [comment]
I have a fix for this, can create a PR shortly (today).  

Additionally, we should document the EXCLUDE_FILE variable, because it can be used for another use case, that of someone wanting to test the reinclusion of excluded tests without changing the existing exclude file.

**Comment by smlambert** [comment]
FIxed by #1821
